benchmark/README.md

```diff
@@ -78,7 +78,7 @@ You must be in the repo root directory to run these commands.
 Run the default benchmark suites:
 
 ```
-$ ./bench.sh --quick
+$ bench.sh --quick
```

Comment:
`bin/bench.sh`?
Is it symlinked?

Reply:
The previous statement says `You must be in the repo root directory to run these commands.`
You should be in the benchmarks directory?